packages/cli/config/eslint-factory.js

```diff
@@ -165,6 +165,23 @@ function createConfig(dir, extraConfig = {}) {
           ],
         },
       },
+      {
+        files: ['**/src/generated/**/*.ts'],
+        rules: {
+          ...tsRules,
+          'no-unused-vars': 'off',
+          'unused-imports/no-unused-imports': 'error',
+          'unused-imports/no-unused-vars': [
```

Comment:
Hold up, why are these added here? `@typescript-eslint` is what we use for unused vars. In general I feel there's really no point in enabling any additional rules for generated code?

Reply:
Ah so it trims away things to make `tsc` work? Makes sense then, let's :shipit: